conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -216,7 +220,7 @@ object DynamicConformanceJob {
     val publishDirSize = fsUtils.getDirectorySize(publishPath)
     performance.finishMeasurement(publishDirSize, recordCount)
     PerformanceMetricTools.addPerformanceMetricsToAtumMetadata(spark, "conform",
-      stdPath, publishPath, EnceladusRestDAO.userName)
+      stdPath, publishPath, EnceladusRestDAO.userName, cmdLineArgss)
```

Comment:
```suggestion
      stdPath, publishPath, EnceladusRestDAO.userName, cmdLineArgs)
```
Sorry, build fails here.